config/routes.rb

```diff
@@ -3,6 +3,7 @@
     post 'submit', to: 'ideas#submit'
   end
   devise_for :users
+  get 'users/list', to: 'users#list'
```

Comment:
See other comment about using :index, then this could be resources as you initially tried